governance/xc_admin/packages/xc_admin_common/src/propose.ts

```diff
@@ -409,6 +409,7 @@ export class MultisigVault {
             needToFetchBlockhash = true;
           }
           console.log(e);
+          await new Promise((r) => setTimeout(r, 1000));
```

Comment:
I'd suggest increasing this to 5 seconds, we are not in a hurry and 1 second may not be enough sometimes.

Reply:
mmm let's try 5 seconds if we still struggle

Reply:
yes

Reply:
Made it 3 seconds for now